docs/source/en/main_classes/pipelines.md

```diff
@@ -302,6 +302,56 @@ That should enable you to do all the custom code you want.
 
 [Implementing a new pipeline](../add_new_pipeline)
 
+## Integration with PEFT library
+
+You can use adapter models that have been trained using the PEFT library with the pipelines. Currently the supported tasks for adapters are:
+
+- `text-generation`
+- `text2text-generation`
+- `question-answering`
+- `token-classification`
+- `feature-extraction`
+
+Make sure to first install the PEFT library:
+
+```bash
+pip install peft
+```
+and double check that you have passed a valid Hub model name with PEFT adapters:
+```python
+from transformers import pipeline
+
+model_id = "ybelkada/opt-350m-lora"
+pipe = pipeline("text-generation", model=model_id)
+
+output = pipe("Hello")
+```
+Or a `PeftModel` to the pipeline:
+```python
+from transformers import pipeline, AutoTokenizer
+from peft import AutoPeftModelForCausalLM
+
+model_id = "ybelkada/opt-350m-lora"
+
+tokenizer = AutoTokenizer.from_pretrained(model_id)
+peft_model = AutoPeftModelForCausalLM.from_pretrained(model_id)
+
+pipe = pipeline("text-generation", peft_model, tokenizer=tokenizer)
```

Comment:
```suggestion
pipe = pipeline(task="text-generation", model=peft_model, tokenizer=tokenizer)
```
Just makes it more readable to newcomers IMO (even I always write it this way to make sure I'm using the right args, as order is never in my mind)